docs/cms/input-types.adoc

```diff
@@ -91,22 +91,42 @@ image::Content-selector-default.png[An input box with a dropdown options below i
   </config>
 </input>
 ----
-<1> *allowContentType* is used to limit the content types that may be selected using the <<allowContentType Samples>>
+<1> *allowContentType* is used to limit the content types that may be selected using the <<allowContentType>>
 <2> *allowPath* is used to limit the path of the content that may be selected using the <<allowPath Samples>>. By default, Content Selector
 only displays content from the same site.
 <3> *treeMode* toggles between tree and flat modes. Default value is `false` (flat mode). +
 image:Content-selector-tree.png[Input box with a dropdown with a tree structure, 350]
 <4> *hideToggleIcon* hides the toggle icon when set to `false`. +
 image:Content-selector-no-icon.png[Input box without an icon, 500]
 
-=== allowContentType Samples
+[#allowContentType]
+=== Allow content type pattern
+There are two modes of matching content type patterns in XP: `LEGACY` to simulate pattern matching before image:../images/xp-770.svg[XP 7.7.0,opts=inline] and `MATCH` as current default mode. It can be configured in <<../deployment/config.adoc#admin, config>>.
 
+==== Legacy +
+
+If pattern value can be found in any part of the content type name then it is accepted.
+Pattern will be extended by current application name if it's not set in <application-name:content-name> format or if none of ':', '*' symbols were used in pattern or pattern starts with '${app}'.
```

Comment:
I accept your suggestion because it is more clear. it is slightly redundant, because `application-name:content-name` already contains `:`, thus first bullet is covered by second, but I guess it is fine

Reply:
`+\{}[]()` don't' do anything special in LEGACY mode. This is to preserve backwards compatibility as much as possible

Reply:
Nice. Is there any way it could _not_ be covered? Like if the app name contains `:` for some other reason? I'm guessing you're not allowed to use that character in app names, though 💁🏾